libs/ui/design-system/src/lib/textarea.tsx

```diff
@@ -1,8 +1,23 @@
 import { Label } from '@nmit-coursition/ui/primitives/label'
-import { Textarea as TextareaPrimitive } from '@nmit-coursition/ui/primitives/textarea'
+import * as TextareaPrimitive from '@nmit-coursition/ui/primitives/textarea'
 import { cn } from '@nmit-coursition/ui/utils'
+import { type VariantProps, cva } from 'class-variance-authority'
 
-interface TextareaWithTextProps extends React.ComponentProps<'textarea'> {
+const textAreaVariants = cva('', {
+  variants: {
+    variant: {
+      primary: '',
+      invalid: '',
```

Comment:
**suggestion:** No invalid or valid, it's based on validity of input/textarea.

Reply:
Ookay

Reply:
I'm creating an API for validations.